spec/index.bs

```diff
@@ -12768,6 +12768,12 @@ instance by passing the string literal `'webgpu'` as its `contextType` argument.
     </pre>
 </div>
 
+Unlike WebGL or 2D context creation, The second argument of
+{{HTMLCanvasElement/getContext()}}, |context attributes| for context configuration,
+is ignored. {{GPUCanvasContext/configure()}} should be used instead.
+
+Note: The user agent may show a warning if |context attributes| are provided.
```

Comment:
nit: Variables only work properly inside algorithms; without an algorithm, it appears as a clickable link but clicking does nothing. I'll tweak some wording and fix this too.